styles/globals.css

```
@@ -1,6 +1,151 @@
+  --transition-duration-normal: 200ms;
+  --transition-duration-slow: 300ms;
+  --transition-duration-slower: 500ms;
+  --ease-bounce-subtle: cubic-bezier(0.34, 1.56, 0.64, 1);
+  --ease-smooth: cubic-bezier(0.4, 0, 0.2, 1);
+  --ease-out-back: cubic-bezier(0.34, 1.56, 0.64, 1);
+
+  /* === ANIMATIONS === */
+  --animate-fast-pulse: fast-pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;
+  --animate-appear: appear 0.3s ease-out;
+  --animate-disappear: appear 0.3s ease-in reverse forwards;
+  --animate-heartBeat: heartBeat 300ms ease-out;
```

Stylelint (17.11.0)

[error] Expected "heartBeat" to be "heartbeat" (value-keyword-case)

(value-keyword-case)